messages/referral_converter.py

```diff
@@ -0,0 +1,25 @@
+from google.appengine.ext import ndb
+
+from domain.institution_domain import Institution
+from messages.referral_messages import ReferralResponse
+
+
+def convert_entity_into_to(referral):
+    return ReferralResponse(claim_number=referral.claim_number,
+                            name=referral.name,
+                            email=referral.email,
+                            phone=referral.phone,
+                            consent=referral.consent,
+                            institution=Institution.encode(referral.institution_id.get()))
+
+
+def convert_to_into_entity(referral_to):
+    # @Marcot, this could be a problem as there are cross reference.
+    from domain.referral_domain import Referral
```

Comment:
Uh, it seems weird to do an import in the middle of a function. I didn't think you could actually do this.

Reply:
Hahaha!

This is UGLY, but to make the code compile, this was the only way I could do as there is currently a "cross-dependency problem".